modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,39 @@
+(link:https://issues.redhat.com/browse/LOG-3548[LOG-3548])
+
+* With this update, The Loki Operator provides `PodDisruptionBudget` resources for LokiStack that determines the minimum number of pods that must be available per component to ensure normal operations. (link:https://issues.redhat.com/browse/LOG-3839[LOG-3839])
+
+* With this update, you can control the Loki component on cluster nodes to improve performance and resilience by applying a set of default affinity and anti-affinity policies.
+(link:https://issues.redhat.com/browse/LOG-3840[LOG-3840])
+
+* With this update, you can deploy multiple, isolated, and RBAC-protected `ClusterLogForwarder` custom resource (CR) instances in any namespace. This allows independent groups to forward desired logs to any destination and also separates logging groups to interfere with another configuration. (link:https://issues.redhat.com/browse/LOG-1343[LOG-1343])
+
+* With this update, you can manage zone-aware data replication as an administrator in LokiStack, in order to enhance reliability and also store logs in the event of a zone failure. (link:https://issues.redhat.com/browse/LOG-3266[LOG-3266])
+
+* With this update, you can enable the Logging Console Plugin when Elasticsearch is the default Log Store. (link:https://issues.redhat.com/browse/LOG-3856[LOG-3856])
+
+* With this update, you can use the flow control or rate limiting mechanism to limit the volume of log data that can be collected or forwarded by dropping excess log records. The input limits prevent badly-behaved containers from overloading the {logging-uc} and the output limits put a ceiling on the amount of log data stored. (link:https://issues.redhat.com/browse/LOG-884[LOG-884])
+
+* With this update, you can select a Loki deployment size based on log volume and expected query usage, in order to properly size the infrastructure nodes pool. You can manage an entry-level or a high-scale LokiStack setups for your {product-title} clusters. (link:https://issues.redhat.com/browse/LOG-4329[LOG-4329])
+
+* With this update, you can configure the log collector to look for HTTP connections and receive logs as an HTTP server, also known as a webhook. (link:https://issues.redhat.com/browse/LOG-4562[LOG-4562])
+
+* With this update, you can forward Kubernetes and OpenShift API server events in accordance with audit logging policies. The policies are now associated with audit log inputs in the `ClusterLogForwarder`. (link:https://issues.redhat.com/browse/LOG-3982[LOG-3982])
+
+[id="logging-release-notes-5-8-0-known-issues"]
+== Known Issues
+* Currently, there is a flaw in handling multiplexed streams in the HTTP/2 protocol where you can repeatedly make a request for a new multiplex stream and immediately send an `RST_STREAM` frame to cancel it. This created extra work for the server setting up and tearing down the streams, resulting in a denial of service due to server resource consumption.
```

Comment:
Can you add the below bugs as known issue.
```
https://issues.redhat.com/browse/LOG-4706
https://issues.redhat.com/browse/LOG-4403
https://issues.redhat.com/browse/LOG-3933
```

Reply:
Sure! On it.

Reply:
@anpingli As discussed, will need some more information on the bugs to document as Known Issue. I have also included both the errata links in the PR. Let me know if that looks okay to you.

Reply:
@cahartma / @jcantrill / @Clee2691 - Please provide text for the above Known Issues, using this format: 

```
There is currently a known issue <present tense explanation> under <X conditions>.

Workaround: <workaround>.
```